public_html/wp-content/mu-plugins/4-helpers-wcpt.php

```diff
@@ -51,7 +75,7 @@ function get_wordcamp_post( $site_id = null ) {
 	switch_to_blog( BLOG_ID_CURRENT_SITE );
```

Comment:
Updating the comment for this line would be nice too, since it can also refer to the Pilot Events root site.

_( That's assuming that the new CPT in #911 will be activated on `events.wordpress.org` rather than `central.wordcamp.org` )_

Reply:
See https://github.com/WordPress/wordcamp.org/pull/937/files#r1260469532